src/common/proxy_wasm/ngx_proxy_wasm_foreign_callback.h

```diff
@@ -0,0 +1,29 @@
+#ifndef _NGX_PROXY_WASM_FOREIGN_CALLBACK_H_INCLUDED_
+#define _NGX_PROXY_WASM_FOREIGN_CALLBACK_H_INCLUDED_
+
+
+#include <ngx_wasm.h>
+#include <ngx_proxy_wasm.h>
+#include <ngx_wasm_subsystem.h>
+
+
+struct ngx_proxy_wasm_foreign_cb_s {
+    ngx_queue_t                         q;
+    ngx_proxy_wasm_exec_t              *pwexec;
+#if (NGX_WASM_HTTP)
+    ngx_http_wasm_req_ctx_t            *rctx;
```

Comment:
```suggestion
```

Reply:
Keeping `rctx` as it's not present in `ngx_proxy_wasm_exec_t` or `ngx_proxy_wasm_ctx_t`.

Reply:
Ah, thanks for clarifying!

Reply:
Ah right! So that settles it, sounds good then!